src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.vue

```diff
@@ -65,27 +65,10 @@
   import Lang from '@/lib/filters/lang'
   import {namespace} from 'vuex-class'
   import {BlockConfigFieldType} from '@/store/flow/block'
+  import {OutputBranchingType} from '@/components/interaction-designer/block-editors/BlockOutputBranchingConfig.model'
 
   const flowVuexNamespace = namespace('flow')
 
-  export enum OutputBranchingType {
```

Comment:
moved this inside a ts file `src/components/interaction-designer/block-editors/BlockOutputBranchingConfig.model.ts` to avoid something like ` warning  in ./src/store/flow/block-types/index.ts "export 'IContactPropertyOption' was not found in './Core_SetContactPropertyStore'`